package/polysat/Config.in.host

```diff
@@ -0,0 +1,16 @@
+source "package/polysat/arsftp/Config.in.host"
```

Comment:
This is a separate change for adding host builds of our util programs. Not related to the 18.04 migration